<?xml version="1.0" encoding="UTF-8"?><testrun name="TestUpload" project="HDFSdemo" tests="5" started="5" failures="0" errors="1" ignored="0">
  <testsuite name="hadoop.TestUpload" time="2.269">
    <testcase name="testDataNode" classname="hadoop.TestUpload" time="1.431"/>
    <testcase name="testMkdir" classname="hadoop.TestUpload" time="0.455"/>
    <testcase name="testFileBlockLocation" classname="hadoop.TestUpload" time="0.023"/>
    <testcase name="testMkdir4" classname="hadoop.TestUpload" time="0.255"/>
    <testcase name="testDownload" classname="hadoop.TestUpload" time="0.105">
      <error>java.io.FileNotFoundException: C:\Users\xinyu\Dropbox\gg\Job_hunting\big data\BigData\2.Hadoop\tmp\b.jar (The system cannot find the path specified)&#13;
	at java.base/java.io.FileOutputStream.open0(Native Method)&#13;
	at java.base/java.io.FileOutputStream.open(Unknown Source)&#13;
	at java.base/java.io.FileOutputStream.&lt;init&gt;(Unknown Source)&#13;
	at java.base/java.io.FileOutputStream.&lt;init&gt;(Unknown Source)&#13;
	at hadoop.TestUpload.testDownload(TestUpload.java:61)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:538)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:760)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:460)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:206)&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
